data/json/hobbies.json

```
@@ -170,42 +180,127 @@
+    "points": 1,
+    "skills": [ { "level": 2, "name": "swimming" } ]
+  },
   {
     "type": "profession",
     "subtype": "hobby",
     "id": "swimming",
-    "name": "Swimming",
+    "name": "Swimming (Intermediate)",
     "description": "You're a strong swimmer, and you're pretty sure zombies aren't.  You can't swim forever, but it's certainly an advantage over the undead.",
     "points": 2,
     "skills": [ { "level": 4, "name": "swimming" } ],
     "traits": [ "OUTDOORSMAN" ]
   },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "swimming_expert",
+    "name": "Swimming (Expert)",
+    "description": "You're an excellent swimmer and may be part fish.  Plus, all that swimming has left you in great shape.  There aren't any zombie sharks in the water, right?",
+    "points": 5,
+    "skills": [ { "level": 6, "name": "swimming" } ],
+    "traits": [ "OUTDOORSMAN", "GOODCARDIO" ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "baseball_beginner",
+    "name": "Baseball (Beginner)",
+    "description": "You used to play baseball, occasionally.  Maybe you were on your school team growing up, but it was never a focus of yours.  Hopefully you can still remember how to use a bat when your life depends on it.",
+    "points": 2,
+    "skills": [ { "level": 2, "name": "bashing" }, { "level": 2, "name": "throw" } ]
+  },
   {
     "type": "profession",
     "subtype": "hobby",
     "id": "baseball",
-    "name": "Baseball",
+    "name": "Baseball (Intermediate)",
     "description": "You got the stance, you got the grip, you got the game.  Swinging at a zombie's head is close enough to swinging at a baseball… right?",
-    "points": 3,
-    "skills": [ { "level": 3, "name": "bashing" }, { "level": 2, "name": "throw" }, { "level": 1, "name": "swimming" } ]
+    "points": 4,
+    "skills": [ { "level": 4, "name": "bashing" }, { "level": 4, "name": "throw" }, { "level": 1, "name": "swimming" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "baseball_pitcher",
+    "name": "Baseball Pitcher (Expert)",
+    "description": "You're accustomed to striking out poor batters at the plate and can probably throw a mean knuckleball.  Now you get to practice your fastball, but perhaps with rocks or grenades this time.",
+    "points": 6,
+    "skills": [ { "level": 4, "name": "bashing" }, { "level": 7, "name": "throw" }, { "level": 3, "name": "swimming" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "baseball_expert",
+    "name": "Baseball (Expert)",
+    "description": "You didn't quite make it into the big leagues, but you were close.  Now it's time to go for the grand slam or you'll be dealing with worse than a strike out.",
+    "points": 6,
+    "skills": [ { "level": 6, "name": "bashing" }, { "level": 5, "name": "throw" }, { "level": 3, "name": "swimming" } ],
```

```suggestion
    "skills": [ { "level": 6, "name": "bashing" }, { "level": 5, "name": "throw" } ],
```
I guess you forgot to remove the swimming skill? I don't see why a baseball_expert would have swimming 3

---

I remind you that `swimming` is id of athletics skill

---

As per Guardian, "swimming" is the id of Athletics, and I gave sporty backgrounds some athletics skill because, well, they're athletes.